ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -85,12 +85,14 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "http_2xx_failed_requests
   query = <<-QUERY
 requests
 ${local.skip_on_weekends}
-| where toint(resultCode) between (200 .. 299) and success == false and timestamp >= ago(5m)
+| where toint(resultCode) between (200 .. 299) and timestamp >= ago(5m)
+| sort by timestamp asc
+| summarize failureRate = iff((todouble(sumif(1, success == false)) * 100 / todouble(sumif(1, success == true)) >= ${var.http_2xx_failure_rate_threshold}), 1, 0)
   QUERY
 
   trigger {
-    operator  = "GreaterThan"
-    threshold = var.failed_http_2xx_threshold
+    operator  = "Equal"
+    threshold = 1
```

Comment:
The threshold needs to be a whole number. I've set it as a hardcoded binary value. The query returns 1 for when we are greater than or equal to our proper threshold and zero when we're less than.

I've also set our failure rate threshold to 1, which is 1 percent of total requests. I've settled on 1 percent based on the charts I included in the description.